src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="post"
+      data-qa="big"
+      class="searchForm"
+    >
+      <label for="search-input" class="lableClass">
```

Comment:
i'd use a more descriptive classname
```suggestion
      <label for="search-input" class="searchbar__label">
```